sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/administration/DocumentModelAdminAsyncClientJavaDocCodeSnippets.java

```diff
@@ -58,13 +58,12 @@ public void createDocumentTrainingAsyncClientWithPipeline() {
     }
 
     /**
-     * Code snippet for {@link DocumentModelAdministrationAsyncClient#beginBuildModel(String, DocumentBuildMode, String)}
+     * Code snippet for {@link DocumentModelAdministrationAsyncClient#beginBuildModel(String, DocumentBuildMode)}
      */
     public void beginBuildModel() {
-        // BEGIN: com.azure.ai.formrecognizer.administration.DocumentModelAdministrationAsyncClient.beginBuildModel#String-String
+        // BEGIN: com.azure.ai.formrecognizer.administration.DocumentModelAdministrationAsyncClient.beginBuildModel#String-DocumentBuildMode
```

Comment:
just trying to understand since I see this update all over the PR, was DocumentBuildMode just missing from before / is now replacing the removed modelId?

Reply:
yes, I think it was missing from the codesnippet tag, but was included in the examples, so it still showed up on the code examples.
Just corrected this.